.travis.yml

```diff
@@ -30,7 +30,7 @@ env:
         - MAIN_CMD='python setup.py'
         - SETUP_CMD='test'
         - EVENT_TYPE='pull_request push'
-        - PIP_DEPENDENCIES=''
+        - PIP_DEPENDENCIES='gwcs'
```

Comment:
Here's the failure:
https://travis-ci.org/astropy/photutils/jobs/209157199#L456

Is gwcs in the `astropy-ci-extras` channel?